### ISSUE TITLE
`anovos.feature_recommender.feature_recommender.init_input_fer()` requires internet access

### ISSUE THREAD

**Issue by ionicsolutions** [anovos/anovos#106]
## Expected Behavior 

I can import and use _Anovos_ locally. A specific version of _Anovos_ always behaves the same.

## Current Behavior

`init_input_fer()` downloads a CSV file from GitHub every time it is called:

```python
def init_input_fer():
    """

    Returns
    -------
    Loading the Feature Explorer and Recommender (FER) Input DataFrame (FER corpus)
    """
    input_path_fer = "https://raw.githubusercontent.com/anovos/anovos/main/data/feature_recommender/flatten_fr_db.csv"
    df_input_fer = pd.read_csv(input_path_fer)
    return df_input_fer
```

Also, if this file is ever renamed on or removed from the current `main` branch, old versions of _Anovos_ no longer work. In addition, if the file gets updated, the behavior of _Anovos_ changes in a way that is really difficult for users to track down.

## Steps to reproduce

N/A

### Specifications

- Version: current `main` state
- Platform: Ubuntu Linux (doesn't matter)
- Subsystem:

## Possible Solution

Package the feature explorer input data with the library.

**Comment by kajanansangar** [comment]
@ionicsolutions 
We did not include the corpus (flatten_fr_db.csv) under the package (/src) considering the potential size issues. We hope to grow this corpus drastically in future and it might impact the size of the Anovos GitHub repo & Anovos package. In future, we are planning to host this corpus in our backend systems. Queries can be made via APIs. 

We want to get the User experience with this module first before making any breaking changes.

**Comment by ionicsolutions** [comment]
The file is just 130KB, I don't see any issue with including it in the package. To install _Anovos_, users need to download hundreds of MB of dependencies anyway.

The problem with the current solution is mostly that, as I sketched out, _Anovos_ will suddenly break if the file is ever removed from the main branch or renamed.

Further, it is a nightmare for users if _Anovos_ is working fine for them and then suddenly (because someone updated the file on GitHub) the behavior changes. Any part of a library should be fixed/versioned and available permanently.

**Comment by kajanansangar** [comment]
ok assigning this to @dattranm look into this

**Comment by dattranm** [comment]
Thanks @ionicsolutions . This issue is addressed in #107 , and in there, I include the data file inside the package as suggested. Can you check and lmk if anything?
